sklearn/utils/validation.py

```diff
@@ -1655,6 +1655,50 @@ def check_is_fitted(estimator, attributes=None, *, msg=None, all_or_any=all):
         raise NotFittedError(msg % {"name": type(estimator).__name__})
 
 
+def _estimator_has(attr, delegates=None):
+    """Check if we can delegate a method to the underlying estimator.
+
+    We check the `delegates` in the order they are passed.
+    By default, we first check the fitted estimator if available,
+    otherwise we check the unfitted estimator.
+
+    getattr() returns True if the attribute exists in the object,
+    and raises `AttributeError` if it doesn't.
+
+    If none of the delegates are present in the object, ValueError is raised.
+    Parameters
```

Comment:
```suggestion

    Parameters
```

Just adding an empty line to adhere to the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html) which is enforced by CI for all the tools provided via API. Here, it is not so important, but let's keep the scheme. :)